tier4/AutowareArchitectureProposal.iv
# Ros2 v0.8.0 autoware vehicle rviz plugin

# How to check

1. Open rviz and subscribe topic from display type: {A. ConsoleMeter; B. SteeringAngle; C. TurnSignal}

2. publish topic and confirm rviz

A.
```
ros2 topic pub /tmp autoware_vehicle_msgs/msg/Steering '{header:{frame_id: "base_link"}, data: 1.0}' 
```

B.
```
ros2 topic pub /tmp2 geometry_msgs/msg/TwistStamped '{header:{frame_id: "base_link"}, twist:{linear:{x: 1.0}}}'
```

C. 
```
$ros2 topic pub /tmp3 autoware_vehicle_msgs/msg/TurnSignal '{header:{frame_id: "base_link"}, data: 2}'
```
